src/control/controllers/PidTwoAxesController.cpp

```diff
@@ -0,0 +1,36 @@
+//
+// Created by ratoone on 25-09-19.
+//
+
+#include <control/controllers/PidTwoAxesController.h>
+
+/**
+ * Create a PID controller for two axes, with different parameters
+ * @param pidXParams a tuple with 4 parameters for X dir PID - proportional, integrative, derivative and feedforward gain
+ * @param pidYParams a tuple with 4 parameters for Y dir PID - proportional, integrative, derivative and feedforward gain
+ */
+PidTwoAxesController::PidTwoAxesController(std::tuple<double, double, double, double> pidXParams, std::tuple<double, double, double, double> pidYParams){
+    pidXAxis = PidController(pidXParams);
+    pidYAxis = PidController(pidYParams);
+}
+
+/**
+ * Computes the PID output for the two axes
```

Comment:
What is PID?

Reply:
https://en.m.wikipedia.org/wiki/PID_controller

Reply:
If this URL clearly explains how it works then I propose to add this URL to the documentation.